### PR TITLE
Add input/output tensor names

### PR DESCRIPTION
**Pull request by shizy818**

### CODE REVIEW

**Comment by shizy818** [comment]
Hi @caoimhinuibrian, I submit this PR about the signature function:
1. get input/output tensor names
2. When I compile a model with unsigned integer input, I get following example:
```
[ {"type": uint32, "dims": [2, 3]} ]
```
However it cannot be parsed as a json. So I replace all types as string. Our application need type `float`, `integer` and `long` and I will handle it on our side.

Please help to review. Thanks!

File: src/Builder/FrontendDialectTransformer.cpp [hunk 1]
```diff
@@ -1085,13 +1085,8 @@ class FrontendGenImpl {
   }
 
   // construct JSON type from the argument type
-  // for example - a 3D array of f32 might produce something like
-  //     {"type" : "float" , "dims" : [4, 256, 16]}
-  // actually, this function would produce
-  //     {"type" : "f32" , "dims" : [4, 256, 16]}
-  //  for this example. The "f32" is mapped to "float" in getSignature, below
-  //
-  void concatTypeString(Type argType, llvm::raw_ostream &dstream) {
```
Comment (gongsu832) [inline src/Builder/FrontendDialectTransformer.cpp:1094]: I would not remove comments. If the comments are no longer valid due to code change, you should update the comments as well.

Comment (AlexandreEichenberger) [reply src/Builder/FrontendDialectTransformer.cpp:1094]: @shizy818 Please keep the comment, updated for the current syntax.

**Review by AlexandreEichenberger — CHANGES_REQUESTED** [review]

**Comment by caoimhinuibrian** [comment]
> Hi @caoimhinuibrian, I submit this PR about the signature function:
> 
> 1. get input/output tensor names
> 2. When I compile a model with unsigned integer input, I get following example:
> 
> ```
> [ {"type": uint32, "dims": [2, 3]} ]
> ```
> 
> However it cannot be parsed as a json. So I replace all types as string. Our application need type `float`, `integer` and `long` and I will handle it on our side.
> 
> Please help to review. Thanks!

@shizy818 don't you want this merged also with the release branch?

File: src/Builder/FrontendDialectTransformer.cpp (hunk 1 above)
Comment (shizy818) [reply src/Builder/FrontendDialectTransformer.cpp:1094]: @AlexandreEichenberger @gongsu832 thanks, I'm going to update it.

Comment (shizy818) [reply src/Builder/FrontendDialectTransformer.cpp:1089]: @gongsu832 Please check the comments update. thx.

**Comment by AlexandreEichenberger** [comment]
Please update the code so that it passes format, thanks

**Comment by shizy818** [comment]
> Please update the code so that it passes format, thanks

@AlexandreEichenberger format done. Thanks!

File: src/Builder/FrontendDialectTransformer.cpp (hunk 1 above)
Comment (gongsu832) [reply src/Builder/FrontendDialectTransformer.cpp:1132]: Comments look fine. Thanks.

**Review by caoimhinuibrian — APPROVED** [review]
LGTM

**Pull request merged**